src/components/CaptureTable.js

```diff
@@ -186,94 +206,136 @@ class CaptureTable extends Component {
+                      sortDirection={orderBy === attr ? order : false}
+                    >
+                      <TableSortLabel
+                        active={orderBy === attr}
+                        direction={orderBy === attr ? order : 'asc'}
+                        onClick={this.createSortHandler(attr)}
+                        disabled={noSort}
+                      >
+                        {label}
+                      </TableSortLabel>
+                    </TableCell>
+                  ))}
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {capturesArray.map((capture) => (
+                  <TableRow
+                    key={capture.id}
+                    onClick={this.createToggleDrawerHandler(capture.id)}
+                    className={classes.tableRow}
                   >
-                    {label}
-                  </TableSortLabel>
-                </TableCell>
-              ))}
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {capturesArray.map((capture) => (
-              <TableRow
-                key={capture.id}
-                onClick={this.createToggleDrawerHandler(capture.id)}
-                className={classes.tableRow}
-              >
-                {columns.map(({ attr, renderer }) => (
-                  <TableCell key={attr}>
-                    {formatCell(capture, attr, renderer)}
-                  </TableCell>
+                    {columns.map(({ attr, renderer }) => (
+                      <TableCell key={attr}>
+                        {formatCell(capture, speciesState, attr, renderer)}
+                      </TableCell>
+                    ))}
+                  </TableRow>
                 ))}
-              </TableRow>
-            ))}
-          </TableBody>
-        </Table>
-        {this.tablePagination()}
-        <Drawer
-          anchor="right"
-          open={this.state.isDetailsPaneOpen}
-          onClose={this.closeDrawer}
-        >
-          <CaptureDetails capture={capture} />
-        </Drawer>
-        <Filter
-          isOpen={true}
-          onSubmit={this.handleFilterSubmit}
-          filter={this.props.filter}
-        />
+              </TableBody>
+            </Table>
+            {this.tablePagination()}
+            <CaptureDetailDialog
+              open={this.state.isDetailsPaneOpen}
+              capture={capture}
+              onClose={this.closeDrawer}
+            />
+          </div>
+        </Grid>
       </div>
     );
   }
 }
 
-const formatCell = (capture, attr, renderer) => {
+const formatCell = (capture, speciesState, attr, renderer) => {
   if (attr === 'id' || attr === 'planterId') {
     return (
       <LinkToWebmap
         value={capture[attr]}
         type={attr === 'id' ? 'tree' : 'user'}
       />
     );
+  } else if (attr === 'speciesId') {
+    return capture[attr] ? speciesState[capture[attr]] : '--';
```

Comment:
A minor point, but this will return `--` if `capture.speciesId===0`, which could be a valid value.